staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -68,15 +69,69 @@ var (
 		},
 		[]string{"result"},
 	)
+
+	authorizationAttemptsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "authorization_attempts_total",
+			Help:           "Counter of authorized attempts.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
+	)
+
+	authorizationLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Name:           "authorization_duration_seconds",
+			Help:           "Authorization duration in seconds broken out by result.",
+			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
+	)
+
+	authorizationDecisionAnnotationCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "authorization_decision_annotations_total",
```

Comment:
Could you please share some lights on the use-case for this metric?

Reply:
> Could you please share some lights on the use-case for this metric?

It is decision label, it seems that the name here is not very accurate, in the latest commit has been modified, its use case in the last comment in the reply I have described, please review, and hope you provide comments to tell me whether it is reasonable